src/routes/retro-certs.js

```diff
@@ -9,6 +9,39 @@ const weeksCompleted = require("../utils/checkFormData");
 function createRouter() {
   const router = Router();
 
+  async function staffViewAuthStatus(postJson, responseJson) {
+    responseJson.status = AUTH_STRINGS.statusCode.userNotFound;
+    const [userRecord] = await Promise.all([
```

Comment:
You don't need Promise.all since we're only waiting on one promise to resolve. `await cosmos.getUserByNameDobSsn(...` is sufficient.